appeals-service-api/src/schemas/fileUpload.js

```diff
@@ -0,0 +1,20 @@
+const { Schema, model } = require('mongoose');
+
+const FileUploadSchema = new Schema(
+  {
+    // uuid: {
+    //   type: String,
+    //   required: true,
+    // },
+    // _id: { type: Schema.Types.ObjectId },
```

Comment:
To have a uuid we either:

* need a way to set this on create on the `appeals-service-api` 

I haven't yet figured out how to do this for a nested resource. 

* Or, we pass this ourselves

This works, but setting the uuid for a file seemingly shouldn't be the concern of the `forms-web-app`. 

`_id: { type: Schema.Types.ObjectId },` was a way of trying to get Mongo to handle all of this magically. Didn't seem to work.

However, neither of these issues are my primary concern atm.